packages/eds-tokens/src/tokens/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
Don't think we need this file now?

Reply:
I don’t think so either, but it did a good job while it lasted. Just hanging there, waiting for some friends. And when they finally arrive it’s time to go. Bummer.